wideq/client.py

```diff
@@ -349,8 +353,26 @@ def enum_name(self, key, value):
         """Look up the friendly enum name for an encoded value.
         """
         options = self.value(key).options
+        if value not in options:
+            logging.warning(
+                'Value `%s` for key `%s` not in options: %s. Values from API: '
+                '%s', value, key, options, self.data['Value'][key]['option'])
+            return _UNKNOWN
```

Comment:
I like the more informative message, but I'm not sure returning a sentinel value is better than crashing with an error, which more forcefully indicates that something has gone wrong… I could imagine things going more _subtly_ wrong if `_UNKNOWN` silently propagates where it shouldn't. Any particular use case you had in mind for this?

Reply:
I'm coming from forking wkd8176's fork of your repo that has washer/dryer support.  I got it to work but it took a lot of trial and error as the Korean washing machine and dryer had enum values that were different then mine and some in that repo were missing from what my washer/dryer reported. 

Because of this it involved updating wideq.py then restarting HA and looking for the next error over and over again.  After restarting HA 20+ times I finally got to a point where the integration worked.  It would have been much easier if the integration worked with the attributes that it knew and logged the rest so that I could fix them all at once instead of fixing one and restarting, etc.

I'm not 100% sure if a warning from the wideq lib would propogate up and show in the HA logs or not, but that was my intention.

Reply:
Got it. I wish there were an easy way to throw on a "development flag" that would catch the relevant KeyError and print a friendly message, but would leave it as a proper error "in deployment"… I can't think of a great way to do that right now, but maybe something will come to me.

Reply:
I did verify the error would be propagated up and show in home assistant.  This would be something helpful a user with a new washer/dryer could paste into a github issue to get it added to the wideq lib.

Reply:
OK! I'm still slightly worried about causing hard-to-diagnose problems by returning sentinel values, and I'm curious if I can find a marginally more elegant solution to this problem… I'll look into it, but after merging this PR if that's cool.

Reply:
Since I was thinking more about home assistant when adding this change, maybe we can log missing enum values and return a sentinel there instead of in the wideq library.